src/app/settings/backendUrl/backendUrl.component.ts

```diff
@@ -18,6 +18,7 @@ export class BackendUrlComponent implements OnInit, OnDestroy {
     'http://localhost:8080'
   ];
   invalidUrl = false;
+  loading = false;
```

Comment:
change name for this variable, this name does not tell what's going on.... maybe `isAlreadySaving` ?